dump_env/cli.py

```diff
@@ -6,6 +6,43 @@
 from dump_env.exceptions import StrictEnvError
 
 
+def needs_quotes(value: str) -> bool:
```

Comment:
it feels like this should be something we can borrow from an ini file generator and/or dotenv itself. If you have a suggestion where we can pull a less naive implementation from, please let me know.

Reply:
@theskumar excuse the summoning, but do you happen to have a suggestion here by any chance, please?